src/map/status.cpp

```diff
@@ -7403,6 +7403,8 @@ static signed short status_calc_hit(struct block_list *bl, status_change *sc, in
 		hit += sc->getSCE(SC_MTF_HITFLEE)->val1;
 	if(sc->getSCE(SC_INCHITRATE))
 		hit += hit * sc->getSCE(SC_INCHITRATE)->val1/100;
+	if (sc->getSCE(SC_POWERUP))
+		hit *= 2; // +100%
```

Comment:
Consider using val1 instead

Reply:
I was wondering about that while developing already. Is it really better to use val1 / val2, etc.?

I feel like those are only needed if the value is different per level, not when the status change has a fixed effect and only the duration is variable.

Because you are basically just either adding redundant code in status.cpp where you set val1/val2, etc. and explain what they mean, even though you could just have it only at the place where SC is actually applied. Or you just set them directly in sc_start but then at the place of the call you wouldn't understand what val1/val2 are for and need to look up the code where they are applied anyway.

After those considerations I decided to not use val1/val2. But I'll bring it up on Discord again before finishing this PR.